lab_1/main.py

```diff
@@ -25,8 +35,14 @@ def remove_stop_words(tokens: list, stop_words: list) -> list:
     stop_words = ['the', 'is']
     --> ['weather', 'sunny', 'man', 'happy']
     """
-    pass
-
+    clean_tokens = []
+    if isinstance(tokens, list) and isinstance(stop_words, list):
```

Comment:
styling*

better write `if not isinstance(tokens, list) and not isinstance(stop_words, list): return []` and then implement function without any nesting